index.html

```diff
@@ -196,7 +196,7 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 					<ul id="project_holder">
 							{% for project in site.data.student_projects %}
 							<li class="student_project">
-								<a href="https://github.com/{{project.github}}">
```

Comment:
@Ploypiti  Why you removed this? Please keep it as it was before

Reply:
I think she did it because her project was not on Github.

Reply:
@ParthS007 I need to add external link, but `https://github.com/` limit to link to a repo only.

Reply:
I recommend you to add another Link rather than Github Link. So it would be fine.